static/css/devhub/base.less

```diff
@@ -0,0 +1,97 @@
+@import '../impala/fonts/fira/fira.less';
+
+// Colors
+@primary-font: 'Fira Sans', sans-serif;;
+
+// Common styles
+section {
+  padding: 0 20px 60px 20px;
+
+  & > h1 {
+    font-size: 40px;
+    line-height: 1.2;
+    margin-bottom: 0;
+    margin-top: 20px;
+  }
+
+  & > p {
+    font-weight: 300;
+    margin-top: 10px;
+  }
+}
+
+.button {
+  padding: 10px 30px;
+  display: inline-block;
+  text-decoration: none;
+  text-align: center;
+  color: #fff;
+  border: 1px #979797;
+  border-radius: 2px;
+  box-shadow: 0 2px 3px 0 rgba(0, 0, 0, 0.3);
+
+  &.green {
+    background-color: #57BD35;
+  }
+}
+
+// Sizes
+
+// Fonts
+* {
+  font-family: @primary-font;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.devhub-header {
+  background-color: #0568BA;
+
+  h1, p {
+    color: #fff;
+  }
+
+  p {
+    font-weight: 300;
+
+    & > a {
+      color: #fff;
+    }
+  }
+}
+
+.devhub-navigation {
+  & ul {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    list-style: none;
```

Comment:
Is this style-sheet inheriting a reset stylesheet? In addons-frontend we are using normalize.css. It would be good to make sure this is using something like that otherwise retro-fitting a reset stylesheet later will probably be more pain.